package.json

```
@@ -34,8 +34,8 @@
     "url": "https://github.com/OpenZeppelin/openzeppelin-confidential-contracts/issues"
   },
   "dependencies": {
-    "openzeppelin-solidity": "file:lib/openzeppelin-contracts",
-    "fhevm": "^0.6.2"
+    "@fhevm/solidity": "0.7.0",
+    "@openzeppelin/contracts": "file:lib/openzeppelin-contracts/contracts"
```

going directly in the contracts repo will prevent us from using js tools in tests. 
```suggestion
    "@openzeppelin/contracts": "file:lib/openzeppelin-contracts"
```

remappings should be used to target the `contracts` subfolder more preciselly in solidity